src/index.css

```
@@ -1,3 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Jua&display=swap');
+*{
+    font-family: 'Jua', sans-serif;
+}
+
```

디자인 다시 봐봤는데 font가 `font-family: Rounded Mplus 1c Bold;`로 바꼈더라구
```suggestion
@import url('https://fonts.googleapis.com/css2?family=Hahmlet:wght@700&family=Noto+Sans+KR:wght@700&display=swap');
*{
    font-family: Rounded Mplus 1c Bold;
}

```
**Add suggestion to batch** 누르면 바로 적용되니까 적용시켜주라